### PR TITLE
Unit tests for SaveBinaryDataJobs added

### PR DESCRIPTION
**Pull request by talisman1234**

### CODE REVIEW

File: src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package com.epam.ta.reportportal.job;
```
Comment (avarabyeu) [inline src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java]: header

File: src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    @Test
+    public void runTestWithContentTypeEqualsImage() throws IOException {
+        byte[] byteArr = {116, 101};
+        when(binaryData.getContentType()).thenReturn(CONTENT_TYPE);
+        when(thumbnailator.createThumbnail(any(byte[].class))).thenReturn(byteArr);
+        when(dataStorageService.saveData(any(BinaryData.class), anyString(), anyMap())).thenReturn("not null");
+        saveBinData.withBinaryData(BIN_DATA).withProject(PROJECT_NAME).withFilename(FILE_NAME).withLog(LOG).run();
+        verify(logRepository, times(1)).save(LOG);
+        verify(dataStorageService, times(2)).saveData(any(BinaryData.class), anyString(), anyMap());
+        }
```
Comment (avarabyeu) [inline src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java]: formatting?

File: src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java (hunk 1 above)
Comment (talisman1234) [inline src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java]: Added header

File: src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java (hunk 2 above)
Comment (talisman1234) [inline src/test/java/com/epam/ta/reportportal/job/SaveBinaryDataJobsTest.java]: Added chaining

**Pull request closed without merging**